### PR TITLE
Refactored view_create to use Twig templates.

### PR DESCRIPTION
**Pull request by udan11**
Before submitting pull request, please check that every commit:

- [X] Has proper Signed-Off-By
- [X] Has commit message which describes it
- [X] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [X] Any new functionality is covered by tests

### CODE REVIEW

File: templates/view_create.twig [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+                    </td>
+                </tr>
+            {% endif %}
+
+            <tr>
+                <td class="nowrap"><label for="algorithm">ALGORITHM</label></td>
+                <td>
+                    <select name="view[algorithm]" id="algorithm">
+                        {% for option in view_algorithm_options %}
+                            <option value="{{ option|e }}"
```
Comment (mauriciofauth) [inline templates/view_create.twig]: You don't need to use the escape filter because it's enabled by default.

Comment (udan11) [inline templates/view_create.twig]: Is it the case for every use of the filter?

I could not find any documentation about this.

Comment (mauriciofauth) [inline templates/view_create.twig]: The `autoescape` [environment option](https://twig.sensiolabs.org/doc/1.x/api.html#environment-options) is enabled by default. So when you do not want to escape, you need to use the [raw filter](https://twig.sensiolabs.org/doc/1.x/filters/raw.html).

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Pull request merged**